wolfcrypt/src/wc_port.c

```diff
@@ -1559,7 +1559,49 @@ int wolfSSL_CryptHwMutexUnLock(void)
     }
 
 #elif defined(WOLFSSL_PTHREADS)
+    #ifdef WOLFSSL_USE_RWLOCK
+        int wc_InitRwLock(wolfSSL_RwLock* m)
+        {
+            if (pthread_rwlock_init(m, 0) == 0)
+                return 0;
+            else
+                return BAD_MUTEX_E;
+        }
+
```

Comment:
NIT: Make the spacing match. One CRLF.

Reply:
Matching. I did this because there is an extra `#ifdef` around these.